test/node/connection-handshake.js

```diff
@@ -7,7 +7,7 @@ const jstp = require('../..');
 const app = require('../fixtures/application');
 const application = new jstp.Application(app.name, app.interfaces);
 
-const Transport = require('../util/transport.mock');
+const Transport = require('../mock/transport.mock');
```

Comment:
I don't think we need to duplicate the word "mock". `../mock/transport` will work just fine.